src/minit-assets.php

```diff
@@ -243,27 +230,30 @@ protected function get_asset_relative_path( $handle ) {
 			return false;
 		}
 
-		$item_url = $this->handler->registered[ $handle ]->src;
+		if ( ! empty( $this->handler->registered[ $handle ]->src ) ) {
+			$item_url = $this->handler->registered[ $handle ]->src;
 
-		if ( empty( $item_url ) ) {
-			return false;
-		}
+			// Inline block scripts are sometimes relative URLs.
+			if ( 0 === strpos( $item_url, '/' ) ) {
```

Comment:
This will return false positives for URLs without protocol, though such as `//example.com/....`